libs/api-interfaces/src/lib/aggregate-views/view-models/resources/song.view-model.interface.ts

```diff
@@ -14,5 +14,5 @@ export interface ISongViewModel extends IBaseViewModel {
 
     lengthMilliseconds: number;
 
-    contributions: string[];
+    contributions: ContributorWithId[];
```

Comment:
I think we forgot to remove this prop. Isn't it inherited from the base class?

Reply:
Updated to extend `IBaseResourceViewModel`